src/modules/auto-flickity.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @author Deux Huit Huit
+ * 
+ * Auto flickity module
+ */
+(function ($, undefined) {
+
+	'use strict';
+	
+	var win = $(window);
+	var site = $('#site');
+	var page = $('.page');
+
+	/*
+		Front-end Integration Hierarchy:
+
+		|- FLICKITY CTN : .js-auto-flickity-slider-ctn
+		|		|- CELL-CTN : .js-auto-flickity-ctn
+		|		|		|- CELL (REPEATED): .js-auto-flickity-item
+		|		|		|
+		|		|- NAV BTNS: .js-auto-flickity-nav-btn
+
+
+		Notes:
+
+		Flickity wont be activated if only one cell(o.cellSelector) is
+		detected. It will add the aborted class(o.abortedCl) on the cell-ctn(o.cellCtn)
+
+		You can have more info on the options of Flickity at
+		http://flickity.metafizzy.co/
+
+	 */
+
+	var o = {
+		sliderCtn: '.js-auto-flickity-slider-ctn',
+		cellCtn:'.js-auto-flickity-ctn',
+		cellSelector: '.js-auto-flickity-item',
+		navBtnSelector:'.js-auto-flickity-nav-btn',
+
+		abortedClass:'is-flickity-cancelled',
+		initedClass:'is-flickity-inited',
+		selectedClass:'is-selected',
+		
+		pageDots: false,
+		prevNextButtons: false,
+		cellAlign: 'left',
+		wrapAround: true
+	};
+
+	var onResize = function () {
+		$(o.cellCtn + '.' + o.initedClass).each(function () {
+			$(this).flickity('resize');
+		});
+	};
+	
+	var pageEnter = function (key, data) {
+		page = $(data.page.key());
+
+		page.find(o.cellCtn + ':not('+ o.initedClass +')').each(function() {
+
+			var t = $(this);
+
+			if (t.find(o.cellSelector).length > 1) {
+				t.flickity(o).addClass(o.initedClass);
+				onResize();
+			} else {
+				t.addClass(o.abortedClass);
+				t.find(o.cellSelector).addClass(o.selectedClass);
+				t.closest(o.sliderCtn).find(o.navBtnSelector).each(function () {
+					$(this).remove();
+				});
+			}
+		});
+
+	};
+
+	var pageLeave = function () {
+		page.find(o.initedClass).each(function(){
+			$(this).flickity('destroy').removeClass(o.initedClass);
+		});
```

Comment:
Best pratice would be to add this line after

```js
page = $();
```

This would prevent holding a reference to the DOM Element and prevents recycling of the object.